globus_portal_framework/static/css/v3/search.css

```diff
@@ -0,0 +1,321 @@
+    background-repeat: repeat-x;
+    background-size: cover;
+    height: 120px;
+    position: relative;
+}
+
+.subnav-button {
+    color: white;
+    margin-bottom: 0;
+    margin-top: 50%;
+}
+
+.active {
+    color: #004177 !important;
+    background-color: white !important;
+    font-size: 20px;
+    box-shadow: none !important;
+}
+
+.squared {
+    border-bottom-left-radius: 0 !important;
+    border-bottom-right-radius: 0 !important;
+    border-bottom: 0;
+    background-color: #00264c;
+    font-size: 20px;
+    margin-left: 10px;
+}
+
+.tag {
+    color: white;
+    font-size: 30px;
+    height: 29px;
+    padding-left: 50px;
+    padding-top: 20px;
+    position: relative;
+    top: 35px;
+}
+
+.nav-tabs {
+    height: 120px;
+    border-bottom: 0;
+    padding-bottom: 0;
+}
+
+#globe {
+    height: 50px;
+    width: 50px;
+}
+
+.list-group-item {
+    font-size: 16px;
+    color: #222222;
+}
+
+#results-header {
+    color: #00264c;
+    font-size: 28px;
+}
+
+#datasets-header {
+    color: #777777;
+    font-size: 16px;
+}
+
+#about {
+    font-size: 16px;
+}
+
+#content .card {
+    border: 0;
+    padding-bottom: 25px;
+}
+
+#content .card-header {
+    font-size: 18px;
+    color: #0076bf !important;
+    background-color: white;
+    border-left-color: #22bbff;
+    border-left-width: 3px;
+    border-bottom-color: #cccccc;
+    border-bottom-width: 1px;
+    border-right: 0;
+    border-top: 0;
+    border-style: solid;
+}
+
+#content .card-header a {
+    text-decoration: none;
+    color: #0076bf;
+}
+
+#content .card-body {
+    border-left-width: 3px;
+    border-left-color: #cccccc;
+    border-left-style: solid;
+    border-right-width: 0px;
+    border-top-width: 0px;
+    border-bottom-width: 0px;
+    padding-bottom: 0;
+}
+
+#content .card-body th {
+    color: #777777;
+    font-size: 14px;
+}
+
+#content .card-body td {
+    color: #222222;
+    font-size: 14px;
+}
+
+.table {
+    color: #222222;
+    font-size: 16px;
+}
+
+.fa-mag-glass {
+    content: '';
+    background: url("/static/globus-portal-framework/images/mag_glass.svg");
+    width: 32px;
+    height: 32px;
+    display: inline-block;
+}
+
+.borderless td,
+.borderless th {
+    border: none;
+}
+
+#logo {
+    height: 100%;
+}
+
+#caption-top {
+    caption-side: top;
+}
+
+#details .card-header {
+    font-size: 18px;
+    color: #0076bf !important;
+    background-color: white;
+    border-left-color: #22bbff;
+    border-left-width: 3px;
+    border-bottom-color: #cccccc;
+    border-bottom-width: 1px;
+    border-right: 0;
+    border-top: 0;
+    border-style: solid;
+}
+
+#details {
+    border-top: 0;
+}
+
+#details .card-body {
+    border-left-width: 3px;
+    border-left-color: #cccccc;
+    border-left-style: solid;
+    border-right-width: 0px;
+    border-top-width: 0px;
+    border-bottom-width: 0px;
+    padding-bottom: 0;
+}
+
+#details .card {
+    border: 0;
+    padding-bottom: 25px;
+}
```

Comment:
Like the other static files, could we move this under the `globus-portal-framework` folder?